### ISSUE TITLE
docker run with default config fails on v1.4.1

### ISSUE THREAD

**Issue by alex-m-humai** [pypiserver/pypiserver#343]
Here is the command:

```
$ docker run pypiserver/pypiserver:v1.4.1
Unable to find image 'pypiserver/pypiserver:v1.4.1' locally
v1.4.1: Pulling from pypiserver/pypiserver
df20fa9351a1: Already exists 
36b3adc4ff6f: Already exists 
7eaf559fa1af: Already exists 
ff42851d39f6: Already exists 
8ba142e34a43: Already exists 
7b8d554bdc47: Pull complete 
d68d634278f4: Pull complete 
9aded7aa4123: Pull complete 
c547a38b5626: Pull complete 
Digest: sha256:abd275109b3bfa451d48275b7777e6eee17a0012a60d224b3bffdf9225ef00d3
Status: Downloaded newer image for pypiserver/pypiserver:v1.4.1
Using custom CMD: 
sh: : unknown operand
Error: while trying to list root(/home/pypiserver/packages): [Errno 2] No such file or directory: '/home/pypiserver/packages'

```

this works with v1.3.2

**Comment by elfjes** [comment]
There indeed appears to be an issue with the new docker entrypoint, which sets a default package directory when no arguments are given. Currently the check results in an error. I've created a PR to address this issue

**Comment by mplanchard** [comment]
Thanks for the report, and thanks @elfjes for the fix!

**Comment by mplanchard** [comment]
The fix has been released in v1.4.2. Please let us know if you run into any more issues.
